product-scenarios/backend-service/start_rest_services.sh

```diff
@@ -0,0 +1,190 @@
+
+# check if service restart is forced
+FORCE=$1
+if [ "$FORCE" = "-f" ] || [ "$FORCE" = "--force" ]; then
+    echo "force restarting services"
+    kill_existing_services $PORT1
+    START_REST1=true
+    kill_existing_services $PORT2
+    START_REST2=true
+    kill_existing_services $PORT_STOCK_QUOTE
+    START_STOCK_QUOTE=true
+else
+    verify_service_start $PORT1
+    START_REST1=$?
+    verify_service_start $PORT2
+    START_REST2=$?
+    verify_service_start $PORT_STOCK_QUOTE
+    START_STOCK_QUOTE=$?
+fi
+
+if [[ $START_REST1 -eq 0 ]] || [[ $START_REST2 -eq 0 ]] || [[ $START_STOCK_QUOTE -eq 0 ]]; then
+    #check if git is installed
+    LIST_GIT=$(dpkg -l  git)
+    echo "$LIST_GIT"
+    if [[ $LIST_GIT == "" ]]; then
+      echo "git is not currently installed, hence installing git"
+      apt-get update
+      apt-get install git
+      echo "Successfully installed git"
+    else
+      echo "git is already installed"
+    fi
+
+    git clone --single-branch --branch product-scenarios https://github.com/wso2/product-ei.git
+    echo "cloned branch product-scenarios https://github.com/wso2/product-ei.git"
+
+    if [[ $START_REST1 -eq 0 ]] || [[ $START_REST2 -eq 0 ]]; then
+        #Building the ballerina service
+        cd $BALLERINA_SRC_LOCATION
+        ballerina build
+        echo "Built ballerina service"
+
+        if [[ $START_REST1 -eq 0 ]]; then
+            mkdir -p $REST_SERVICE1_LOCATION
+            cd $REST_SERVICE1_LOCATION
+            cp $BALLERINA_SRC_LOCATION/target/testServices.balx $REST_SERVICE1_LOCATION/
+            cp -rf $BALLERINA_RESOURCE_LOCATION $REST_SERVICE1_LOCATION/
+            nohup ballerina run -e port=$PORT1 $REST_SERVICE1_LOCATION/testServices.balx >> $REST_SERVICE1_LOCATION/service.log &
+            echo "started rest service1 on port: " $PORT1
+        fi
+
+        if [[ $START_REST2 -eq 0 ]]; then
+            mkdir -p $REST_SERVICE2_LOCATION
+            cd $REST_SERVICE2_LOCATION
+            cp $BALLERINA_SRC_LOCATION/target/testServices.balx $REST_SERVICE2_LOCATION/
+            cp -rf $BALLERINA_RESOURCE_LOCATION $REST_SERVICE2_LOCATION/
+            #Modify resource folder in test service2 to contain the loadBalanceMsg with server:2
+            sed -i 's/\"server\"\:1/\"server\"\:2/g' $REST_SERVICE2_LOCATION/resources/json/response/loadBalanceMsg.json
+            nohup ballerina run -e port=$PORT2 $REST_SERVICE2_LOCATION/testServices.balx >> $REST_SERVICE2_LOCATION/service.log &
+            echo "started rest service2 on port: " $PORT2
+        fi
+    fi
+    if [[ $START_STOCK_QUOTE -eq 0 ]]; then
+        #Copying the stockQuote service
+        rm -r $TOMCAT_HOME/webapps/axis2*
+        cp $BACKEND_LOCATION/stockQuoteService/axis2.war $TOMCAT_HOME/webapps/
+        sh $TOMCAT_HOME/bin/startup.sh
+        echo "started stock quote service"
+
+    fi
+fi
+
+echo "############################################# Started Services ##################################################"
+
```

Comment:
Let's schedule cron job at the end after setting up environment

Reply:
done